components/form/rangeDatepicker/src/index.scss

```diff
@@ -19,6 +23,20 @@ $datepicker__selected-color: $bgc-range-datepicker-selected;
     align-items: center;
     display: flex;
     padding-right: $p-xsmall;
+    width: $w-range-datepicker-input;
```

Comment:
Is `!important` necessary in this case ????
Please avoid `!important` and use css weight instead

Reply:
I need !important when i update the fill property